easybuild/easyconfigs/o/OpenMM/OpenMM-7.7.0-foss-2021a.eb

```diff
@@ -0,0 +1,69 @@
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+# Author: Pablo Escobar Lopez
+# sciCORE - University of Basel
+# SIB Swiss Institute of Bioinformatics
+# Update to 7.5.1
+# J. Sassmannshausen / GSTT
+
+easyblock = 'CMakeMake'
+
+name = 'OpenMM'
+version = '7.7.0'
+
+homepage = 'https://openmm.org'
+description = "OpenMM is a toolkit for molecular simulation."
+
+toolchain = {'name': 'foss', 'version': '2021a'}
+toolchainopts = {'opt': True}
+
+source_urls = ['https://github.com/openmm/openmm/archive/']
+sources = ['%(version)s.tar.gz']
+patches = ['%(name)s-7.5.0_fix_path_to_nvcc.patch']
+checksums = [
+    '51970779b8dc639ea192e9c61c67f70189aa294575acb915e14be1670a586c25',  # 7.7.0.tar.gz
+    '050d5b81e70b34d06ef87eafc6d540b2cf716a61f53b1d2a3786504da3ec2e3c',  # OpenMM-7.5.0_fix_path_to_nvcc.patch
+]
+
+builddependencies = [
+    ('CMake', '3.20.1'),
+    ('Doxygen', '1.9.1'),
+]
+
+separate_build_dir = True
+
+dependencies = [
+    ('Python', '3.9.5'),
+    ('SciPy-bundle', '2021.05'),
+    ('CUDA', '11.3.1', '', True),
+    ('SWIG', '4.0.2'),
+]
+
+# Flags for CUDA-enabled build
+configopts = "-DOPENMM_BUILD_CUDA_LIB=ON"
+
+#pretestopts = "export OPENMM_CUDA_COMPILER=nvcc && CTEST_OUTPUT_ON_FAILURE=1"
```

Comment:
Pretty sure this is causing the CI failure, as the `#` is not followed by a space. If this line is no longer relevant, please just take it out (or, if you commented it just for testing purposes, please uncomment it).

Reply:
I updated the PR removing the old `pretestopts` line. Initially I tried with the original `pretestopts` but it failed 1 test: `TestCudaCompiler`, from the OpenMM repo it seems that if you define `OPENM_CUDA_COMPILER=nvcc` will make this particular test to fail https://github.com/openmm/openmm/issues/3373 .